src/InfoBoxes/Content/Other.cpp

```diff
@@ -188,19 +188,13 @@ UpdateInfoBoxNbrSat(InfoBoxData &data) {
     const NMEAInfo &basic = CommonInterface::Basic();
     const GPSState &gps = basic.gps;
 
-    StaticString<80> Temp;
-    StaticString<20> GpsStatus;
-
-    GpsStatus = gettext(GetGPSStatus(basic));
-    data.SetComment(_(GpsStatus));
+    data.SetComment(_(GetGPSStatus(basic)));
     
     if (!basic.alive)
         data.SetComment(_("No GPS"));
-        //    ClearText(NumSat);
     else if (gps.satellites_used_available) {
         // known number of sats
-        Temp.Format(_T("%d"), gps.satellites_used);
-        data.SetValue(Temp);
+        data.FormatValue(_T("%d"), gps.satellites_used);
```

Comment:
`%d` is the wrong format. `satellites_used` is `unsigned`, and thus `%u` should be used. Yes, I know the old code you copied `SystemStatusPanel.cpp` is wrong as well. I'd welcome a fixup patch for that one.

Reply:
ok, i modified my part, the status panel, and the battery infobox, also usinf %d in place of %d.
i compiled, tested. i re-submit.